device-types/HPE/ProCurve-2810-48G.yaml

```diff
@@ -101,11 +106,19 @@ interfaces:
     type: 1000base-t
   - name: '44'
     type: 1000base-t
-  - name: '45'
+  - name: 45-RJ45
```

Comment:
Are these interface names in the config actually suffixed with `-RJ45`? Can you show us an example?
If not, please revert this change and adjust the SFP ones accordingly.